### PR TITLE
update to lodash, remove underscore mixin

### PR DESCRIPTION
**Pull request by wraithgar**
In this branch the only underscore version after an npm i is in jshint.

### CODE REVIEW

**Comment by lukekarrys** [comment]
+1

I do like the idea of removing the util mixin in this case.

**Comment by wraithgar** [comment]
Thanks @lukekarrys I agree that needs to be case-by-case.

**Comment by wraithgar** [comment]
This would obviously be a major version bump due to the removal of the mixin

File: package.json [hunk 1]
```diff
@@ -24,18 +24,20 @@
...
     "jshint": "^2.6.0",
-    "precommit-hook": "^1.0.7",
+    "precommit-hook": "^2.0.1",
     "run-browser": "^2.0.1",
-    "tap-spec": "^2.2.0",
+    "tap-spec": "^3.0.0",
     "tape": "^3.4.0"
   },
+  "files": [
+    "ampersand-paginated-subcollection.js"
```
Comment (HenrikJoreteg) [inline package.json:35]: haven't seen this before, does it mean "only" this file is included? If so, wouldn't we want the readme too?

https://docs.npmjs.com/files/package.json#files

**Comment by HenrikJoreteg** [comment]
other than that question, +1

File: package.json (hunk 1 above)
Comment (wraithgar) [inline package.json:35]: I had the same question when we started doing this.  Turns out readme  is included by default (this is documented _somewhere_ but I can't find it right now), we've been doing this in most of the other repos

**Comment by wraithgar** [comment]
Will publish when #7 merges

**Comment by wraithgar** [comment]
puiblished as `v2.0.0`

**Pull request merged**